Tools/autotest/pysim/vehicleinfo.py

```diff
@@ -92,6 +92,10 @@ def __init__(self):
                 "default_params_filename": ["default_params/copter.parm",
                                             "default_params/gazebo-iris.parm"],
             },
+            "airsim": {
```

Comment:
airsim-copter? to allow other vehicle in future

Reply:
Didn't notice that, I'll change it

Reply:
Done

Reply:
Once we have hexa configuration in Airsim, then this can be changed to airsim-quadx